analysis/stata/active_analyses_failed.R

```diff
@@ -3,13 +3,12 @@
 
```

Comment:
You need to add a line to source specify_paths.R so it can find `path_model_output`.

Reply:
Added